include/swift/AST/DiagnosticsSema.def

```diff
@@ -4485,6 +4485,12 @@ ERROR(actor_inheritance,none,
 ERROR(actor_protocol_illegal_inheritance,none,
       "non-actor type %0 cannot conform to the 'Actor' protocol",
       (DeclName))
+ERROR(distributed_actor_conformance_missing_system_type,none,
+      "distributed actor %0 does not declare ActorSystem it can be used with. "
+      "Declare the associated actor system with a 'typealias ActorSystem = MyActorSystem' "
```

Comment:
I'd suggest you split this into a note + fix-it to insert `typealias ActorSystem = <#Actor System#>` instead another node with talks about the default typealias and puts it right before the declaration.

Reply:
Sounds good, will do